tests/parser/lua.txt

```diff
@@ -91,7 +90,6 @@ return p
 !! test
 must have 3 rows and 3 columns with headers
 !! config
-wgSPARQLEndpoint = "http://blazegraph:8080/bigdata/namespace/kb/sparql"
```

Comment:
This config should be handled by LocalSettings of the MW setup, unless it's for config that affects the outcome of the test.